### DISCUSSION TITLE
Atomic waste

### DISCUSSION THREAD

**Discussion by TmShaz** [electricitymaps/electricitymaps-contrib#3310] [Ideas]
Consider to show atomic waste per MWh to show that e.g. France shifting the problem to the next generations.

**Comment by veqtrus** [comment]
[0.0047 g/kWh](https://www.edfenergy.com/fuel-mix) is negligible compared to 25+ (100+ for most countries).

**Reply by phiphou** [reply]
In France, it's 0.011g/kWh. But 90% of the total weight of them are short lived waste, witch represent 0.1% of the radioactivity. Medium and Long time waste are only 10% of total weight and represent 99.9% of the radioactivity.

**Comment by flacombe** [comment]
That's a good point to discuss.
Negligible or not we must agree on hypothesis to take to consider what is a waste or not.

Countries that operate civil nuclear power plant may have different strategies to process their fission products. Some of them choose to make most of their products a waste and wait a long time for their activity to get low. Other recycle a given part and then reduce their amount of waste.
Among those waste, some are low activity and other are high activity which don't have a comparable life time.
Once those hypothesis properly sourced and documented, time will come to choose what is relevant to show.
Things are a bit more complex than "Nuclear produces waste".

**Comment by jarek** [comment]
> Consider to show atomic waste per MWh to show that e.g. France shifting the problem to the next generations.

Of course, except for countries with vast largely barren areas they can flood for low-methane hydro reservoirs, everyone not using nuclear is also shifting a problem to the next generations.

Now more seriously: How do you think this should be shown? Would you suggest to have a wholly separate mode for only nuclear waste measurements, or to apply some sort of conversion factor into "future pain" that would compare nuclear waste pain vs CO2eq pain?

Electricity Map is good at showing CO2eq/kWh impact. _Everything else_ is an extrapolation. There are numerous ways to reduce a region's electricity use and improve the CO2eq/kWh-electricity figures. Many of them are not great, like for instance shutting down your factories and importing goods produced in areas with even more polluting electricity, or not having an electric train network and using fossil-fueled road transport instead. Using nuclear is another way to have a lower CO2eq/kWh score with some externalities. Unless there's a conversion factor we can use, I'm not really sure how it can be shown on this site.
